test/E2E/HttpTrigger.Tests.ps1

```diff
@@ -14,66 +16,56 @@ Describe 'HttpTrigger Tests' {
         Get-Job -Name FuncJob -ErrorAction SilentlyContinue | Stop-Job | Remove-Job
     }
 
-    It "Test basic HttpTrigger function - Success" -TestCases @(
+    It "Test basic HttpTrigger function - Ok - $FunctionName" -TestCases @(
```

Comment:
Convention is to use `<FunctionName>` instead of `$FunctionName`

Reply:
changed